dev/src/main/java/org/eclipse/codewind/intellij/ui/wizard/AddExistingProjectWizard.java

```diff
@@ -108,16 +112,39 @@ public void run() {
                         String language = subtypeInfo.id;
                         // call validate again with type and subtype hint
                         // allows it to run extension commands if defined for that type and subtype
+                        boolean validationOk = true;
                         if (subtypeInfo != null) {
-                            ProjectUtil.validateProject(name, path, typeInfo.getId() + ":" + subtypeInfo.id, connection.getConid(), indicator);
+                            try {
+                                ProjectUtil.validateProject(name, path, typeInfo.getId() + ":" + subtypeInfo.id, connection.getConid(), indicator);
+                            } catch (Exception e) {
+                                validationOk = false;
+                                CoreUtil.invokeLater(() -> {
+                                    int rc1 = CoreUtil.showYesNoDialog(message("ProjectValidationTitle"), message("ProjectValidationFailedContinueProjectBind", e.getMessage()));
+                                    if (rc1 == Messages.YES) {
+                                        continueWithBind(name, path, language, typeInfo, indicator);
+                                    } else {
+                                        doBind = false;
+                                    }
+                                });
+                            }
+                        }
+                        if (doBind && validationOk) {
+                            continueWithBind(name, path, language, typeInfo, indicator);
                         }
-                        ProjectUtil.bindProject(name, path, language, typeInfo.getId(), connection.getConid(), indicator);
                     }
                 } catch (Exception e) {
                     Logger.log(e);
                 }
             }
         }, message("BindProjectWizardJobLabel", name), false, intellijProject);
+    }
+
+    private void continueWithBind(String name, String path, String language, ProjectTypeInfo typeInfo, ProgressIndicator indicator) {
+        try {
+            ProjectUtil.bindProject(name, path, language, typeInfo.getId(), connection.getConid(), indicator);
+        } catch (Exception e) {
+            Logger.log(e);
```

Comment:
If this bind fails, we should probably display an error to the user.  Also, if the bind fails, the code goes ahead and opens the project in IntelliJ.  I'm not sure that's the right behaviour.

Reply:
Good point.  I was only attempted to address the project validation failures.  I'll add this.

Reply:
Done.  Basic message dialog appears.  Prompt to open project will not occur.